WickedEngine/wiScene.cpp

```diff
@@ -1577,6 +1577,7 @@ namespace wi::scene
 		}
 
 		wi::jobsystem::context ctx;
+		wi::physics::RunPhysicsUpdateSystem(ctx, *this, dt);
```

Comment:
Please double check that this is necessary. I heard about motion state could cause lag, but this line could have larger implications (animation system to physics update for example).